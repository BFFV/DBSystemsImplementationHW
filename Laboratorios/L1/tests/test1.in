5
w text1
w text2
r 1
r 0
f
