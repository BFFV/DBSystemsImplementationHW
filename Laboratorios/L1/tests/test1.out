0
1
text2
text1
